### PR TITLE
fixed mistakes

### PR DESCRIPTION
**Pull request by MarchD**
[DEMO LINK](https://MarchD.github.io/react_movies-list-fetch-movies/)

### CODE REVIEW

**Review by olesiatk — APPROVED** [review]
Great!

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
-        </div>
-        <div className="sidebar">
-          <FindMovie />
-        </div>
+        ) : (
+          <div className="panel-heading">No movies</div>
+        )}
+      </div>
+      <div className="sidebar">
+        <FindMovie onAdd={addNewMovie} error={error} setError={setError} />
```
Comment (aionova) [inline src/App.js:29]:
```suggestion
        <FindMovie
          onAdd={addNewMovie}
          error={error}
          setError={setError}
        />
```

**Review by aionova — COMMENTED** [review]
Good start!

When movie is not found, let's show a text that movie with such title is not found, and not render the preview.
Screenshot 2021-01-29 at 20 26 25

**Review by aionova — CHANGES_REQUESTED** [review]
pls read comments

**Comment by MarchD** [comment]
> Good start!
> 
> When movie is not found, let's show a text that movie with such title is not found, and not render the preview.
> Screenshot 2021-01-29 at 20 26 25

image

It works.)

**Comment by aionova** [comment]
> > Good start!
> > When movie is not found, let's show a text that movie with such title is not found, and not render the preview.
> > Screenshot 2021-01-29 at 20 26 25
> 
> image
> 
> It works.)

Okay)) Got it

**Review by aionova — APPROVED** [review]
Good job!

**Pull request closed without merging**